test/integration/org/pih/warehouse/product/ProductServiceIntegrationTests.groovy

```diff
@@ -327,7 +327,7 @@ class ProductServiceIntegrationTests extends GroovyTestCase {
 
 		// FIXME Export code appends column delimiter for every column (even the last)
 		def expectedHeader = Constants.EXPORT_PRODUCT_COLUMNS.join(",").replace("\n", "") + ","
-		def actualHeader = lines[0]
+		def actualHeader = lines[0].replace("\uFEFF", "")
```

Comment:
This line and the one below it makes me wonder if we need/want a method in `CSVUtils` called something like `stripBOMIfPresent()`?